Cargo.lock

```diff
@@ -2505,9 +2570,9 @@ dependencies = [
  "debugid",
  "encoding_rs",
  "memmap2",
- "minidump-common",
+ "minidump-common 0.22.2",
```

Comment:
Maybe we can bump the prospero dependency here to the same minidump version to not have duplicates.

Reply:
Ok bumped the minidump-writer version and now there is only one of them.